docs/spec/v1beta2/providers.md

```diff
@@ -405,6 +406,51 @@ stringData:
     address: "https://xxx.webhook.office.com/..."
 ```
 
+##### DataDog
+
+When `.spec.type` is set to `datadog`, the controller will send a payload for
+an [Event](events.md#event-structure) to the provided DataDog API [Address](#address).
+
+The Event will be formatted into a [DataDog Event](https://docs.datadoghq.com/api/latest/events/#post-an-event) and sent to the
+API endpoint of the provided DataDog [Address](#address).
+
+The Provider's [Channel](#channel) is used to set the `env` and `kube_cluster_name` tags
```

Comment:
> and kube_cluster_name tags

I guess this is no longer the case after your latest changes. Only the env tag is now set.

Reply:
Thanks, I'll fix this.

Reply:
Thanks! and I just noticed there's also a comment for ` toDataDogTags` that also mentions the `kube_cluster_name` tag.